packages/engine/Source/Scene/BingMapsImageryProvider.js

```diff
@@ -500,6 +500,10 @@ BingMapsImageryProvider.fromUrl = async function (url, options) {
     queryParameters.mapLayer = options.mapLayer;
   }
 
+  if (defined(options.culture)) {
```

Comment:
Please document the new `culture` property under the `BingMapsImageryProvider.ConstructorOptions` type definition.

Reply:
Thanks for your quick review. `culture` property under `BingMapsImageryProvider.ConstructorOptions` type is already documented but it was ineffective. I just fixed it.

https://cesium.com/learn/cesiumjs/ref-doc/BingMapsImageryProvider.html#.ConstructorOptions

Reply:
Fantastic, thanks!